app/models/pieces/pawn.rb

```diff
@@ -1,2 +1,22 @@
 class Pawn < Piece
+  # rubocop:disable all
+  def valid_move?(x_destination, y_destination)
+    # check for Piece valid_move? conditions
+    return false unless super(x_destination, y_destination)
+    # a pawn can never move horizontally or, and never move more than two fields ahead, or more than one sideways
+    return false if moving_horizontally?(y_destination) || diff(x_destination, x_position) > 1 || diff(y_destination, y_position) > 2
+    #a pawn cannot move backwards
+    return false if (player.color == 'white' ? y_destination <= y_position : y_destination >= y_position)
+    # a pawn can only move 2 squares forward from its starting position
+    return false if diff(y_destination, y_position) == 2 && y_position != (player.color == 'white' ? 1 : 6)
+    if diff(x_destination, x_position).zero?
```

Comment:
The x and y diffs are being calculated a lot. Might be best to create an x_diff, y_diff intermediary var